docs/usage/customization-guide.md

```diff
@@ -237,7 +237,9 @@ further be used in label rules specified in
 > while NFD is running. In order to avoid race conditions you should write into
 > a temporary file (outside the `source.d` and `features.d` directories), and,
 > atomically create/update the original file by doing a filesystem move
-> operation.
+> operation. You can also use a dot file like `.my.feature` that contains
+> features, and then automatically update the file name to a real
+> feature file name like `my.feature`.
```

Comment:
Because using another dir can cause issues (if it happens to be on another file system), dot file use is supposed to be **the**  new recommendation, e.g. something like this:

> In order to avoid race conditions you should write into a temporary file , and atomically create/update the original file by doing a file rename operation.  NFD ignores dot files in this directory, so temporary file can be written to the same directory and renamed  (`.my.feature` -> `my.feature`) once file is complete.  Both file names  should (obviously) be unique for the given application.

Reply:
Thanks @eero-t. Updated!